src/reanimated2/animations.js

```diff
@@ -174,7 +174,7 @@ export function withSpring(toValue, userConfig, callback) {
 
       const isOvershooting = () => {
         if (config.overshootClamping && config.stiffness !== 0) {
-          return current < toValue ? current > toValue : current < toValue;
+          return current > toValue;
```

Comment:
What if we use negative velocity? Then we might have current ad positive value but toValue might be negative one.

Then current will immediately be bigger than toValue. Check out clamp of withDecay.

Reply:
Thank you for the explanation! Is the overshootClamping option working for you? I couldn't get it to work in an example. I still struggling to understand how line 177 work.

In my example toValue is 0, let's say current is minus -1: isOvershooting returns false (current < toValue  === true and then current > toValue  === false).

Reply:
I'm not sure it's working right now, can't test right away, but I mean your change wrong here too :)

But I still can't understand how to handle the clamping without velocity. Because velocity is the only thing what can tell the direction of animation.

Reply:
I will do more testing and maybe try to implement it the way it's done in decay and keep you posted. Thank you for your support on this.